dev/TabView/TabView.xaml

```diff
@@ -40,7 +40,8 @@
 
                         <Grid x:Name="TabContainerGrid"
                                 Background="{TemplateBinding Background}"
-                                XYFocusKeyboardNavigation="Enabled">
+                                XYFocusKeyboardNavigation="Enabled"
+                                Margin="0,0,0,-1">
```

Comment:
That fix seems wrong to me. If the contour line (LeftBottomBorderLine, BottomBorderLine, RightBottomBorderLine) is semi-transparent, the TabContentPresenter will show through, no?

Reply:
Agreed, This is also a pixel snapping issue, a negative margin is not appropriate at all scale factors.